Utils/Dataflow/test/utils/reconstruction.py

```diff
@@ -48,7 +78,23 @@ def get_chain_data(es, index, chain_id):
 
 
 def construct_chain(data):
-    ''' Build a chain from the list of its tasks' chain_data. '''
+    ''' Build a chain from the list of its tasks' chain_data.
+
+    Chain is a group of tasks where the first task is the root, and each next
+    task has one of the previous ones as its parent (parent's output includes
+    child's input).
+
+    :param data: chain_data of all tasks in the chain
+    :type data: list
+
+    :return: constructed chain, format is
+             {
+                 ...
+                 taskidN: [taskids of all tasks which are children of taskidN]
```

Comment:
Maybe:
```
taskidN: [ childN1_id, childN2_id, ...]
```
?

Or at least "`child task IDs`"? The shorter things are in the docs, the easier to "scan" them through when trying to figure out out what\`s going on in general -- or to find needed piece of information when need to clarify this or that moment.

Reply:
Done.